net/nft-qos/files/nft-qos.config

```diff
@@ -73,6 +76,15 @@ config default default
 #	option ipaddr 'ABCD:FFED::1/64'
 #	option rate '1024'
 #
+# For Mac address based traffic control Samples :
+#
+# config tcrate
```

Comment:
Silly question but what does `tcrate` mean?
Does it not mac sense to name it `mac`

Reply:
Thanks for the question, **tcrate** means **traffic control rate** -- I changed it to **client** in the new commit. any suggestion for how to do naming in configuration file? Thanks.

Reply:
hmm I read your comment again and I think you are suggesting **mac** instead of **tcrate** or **client**, am I correct?

Reply:
How about **macclient** would this sound better?

Reply:
That was only a hint we could use `tcrate` but we should add a comment to the config what the shortcut stands for.

Reply:
got it thanks, I think **client** should work just fine without extra comment